src/Forum/Content/Discussion.php

```diff
@@ -106,6 +104,7 @@ public function __invoke(Document $document, Request $request)
      */
     protected function getApiDocument(User $actor, array $params)
     {
+        $params['bySlug'] = true;
```

Comment:
Ehm... I am not sure this is wise; this looks a lot like a hard limitation on the code? This would practically make any extension have a hard time to do something different..

Thoughts?

Reply:
Pinging @askvortsov1 on this one.

Reply:
The way I see it, these `Content` classes are where we pre-assemble the page's data in the initial request so we don't need to request that data immediately from the frontend (plus adding SEO tags and all that jazz). So essentially, we're always pinging the API as if we're the frontend, so we should always be requesting stuff by slug (since we don't have any other info than just the URL). What extensions could this limit?